cluster-http/src/test/scala/akka/cluster/http/management/scaladsl/ClusterHttpManagementRoutesSpec.scala

```diff
@@ -225,9 +228,9 @@ class ClusterHttpManagementRoutesSpec
         doNothing().when(mockedCluster).down(any[Address])
 
         Put(s"/cluster/members/$address", urlEncodedForm) ~> ClusterHttpManagementRoutes(mockedCluster) ~> check {
+          status shouldEqual StatusCodes.OK
```

Comment:
Should this now return OK for addresses that are not found? That's not obvious to me.

Reply:
Oops, bad copy pasta.

Reply:
Thanks for catching. Now fixed.